Documentation/EtcdLease.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+The EtcdLease implement interface `grant` and `revoke` to new a lease and revoke a lease to etcd.
```

Comment:
please add a empty line between title and the content. (also for the following sections)